nautobot_chatops/workers/nautobot.py

```diff
@@ -62,6 +62,17 @@ def prompt_for_vlan(action_id, help_text, dispatcher, filter_type, filter_value_
     return dispatcher.prompt_from_menu(action_id, help_text, choices, offset=menu_offset_value(filter_value_1))
 
 
+def prompt_for_circuit(action_id, help_text, dispatcher, circuits=None, offset=0):
+    """Prompt the user to select a valid circuit from a drop-down menu."""
+    if circuits is None:  # A circuit was not provided; provide list of circuits for user to choose from
+        circuits = Circuit.objects.all().order_by("provider", "cid")
+    if not circuits:  # There were no Circuit objects found; notify user
+        dispatcher.send_error("No circuits were found")
+        return (CommandStatusChoices.STATUS_FAILED, "No circuits were found")
+    choices = [(f"{circuit.provider.name}: {circuit.cid}", circuit.cid) for circuit in circuits]
```

Comment:
Are you good with a slash command that has a different display from the underlying circuit ID?

Reply:
Can you expand on that question?  I'm not understanding what you are asking

Reply:
You have it covered in your example that the `/nautobot get-circuit-connections gtt-224`. But the display that you have would be something like `GTT: gtt-224`.

Reply:
Hi Josh.  That's not what is happening.  The pic in the initial comments shows the shortcut, then the circuit ID, then the output.

I believe this format is good, unless there are objections

![image](https://user-images.githubusercontent.com/13647153/132402732-d2daf587-d4c8-4c3a-a515-88a05012bddd.png)

Reply:
Oh, I see.  In the dropdown selector, I sorted circuit IDs by the provider.  I think that's best way to do it, but am open to suggestions